lxc_ssh.py

```diff
@@ -608,7 +608,7 @@ def _bare_run(self, cmd, in_data, sudoable=True, checkrc=True):
         # only when using ssh. Otherwise we can send initial data straightaway.
 
         state = states.index('ready_to_send')
-        if b'ssh' in cmd:
+        if b'ssh' in cmd and self.lxc_version != 2:
```

Comment:
If that is for LXC version one, why don't you specifically compare for "1", instead of excluding anything which is not 2? If there is a LXC version 3, it will likely work more like v2, not v1, but your check here will exclude this version.